config/typescript/@types/mnemonist/index.d.ts

```diff
@@ -0,0 +1,23 @@
+declare module "mnemonist/heap" {
```

Comment:
I guess this is because mnemonist types are wrong. Maybe it's worth it opening a PR fixing them in their repo.

Reply:
Yes, that's the reason. Good point.